django_airavata/templates/base.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>{% block title %}Django Airavata Gateway{% endblock %}</title>
```

Comment:
Can you undo the changes to this file? We do need something like this but I need to think about how to do this for the CIPRES sharing portal.